pkcs11/pkcs11.go

```diff
@@ -77,6 +77,23 @@ func findObjects(context PKCS11Ctx, session p11.SessionHandle, template []*p11.A
 	return objs, nil
 }
 
+func findSlotNumber(context PKCS11Ctx, tokenLabel string) (uint, error) {
+	slots, err := context.GetSlotList(true)
+	if err != nil {
+		return 0, err
+	}
+	for _, slot := range slots {
+		tokenInfo, err := context.GetTokenInfo(slot)
+		if err != nil {
+			return 0, err
+		}
+		if tokenInfo.Label == tokenLabel {
+			return slot, nil
+		}
+	}
+	return 0, errors.New("slot not found")
+}
+
```

Comment:
Can you also add the tokenLabel to the KeyInfo struct below as each tokenLabel is associated with a specific key.

Reply:
Thanks review! I added.